README.md

```diff
@@ -8,5 +8,92 @@ Low-level R library bindings
 
 [API Documentation](https://extendr.github.io/libR-sys/master/libR_sys/index.html)
 
+## Installation
 
+To build this library and test that everything is working as expected, run the following two commands in the library's top-level directory:
 
+```bash
+cargo build
+cargo test
+```
+
+The build script has the following two dependencies:
+
+1. [R](https://cran.r-project.org/): It needs to be installed and available in the search path.
+2. [libclang](https://clang.llvm.org/docs/Tooling.html): Depending on your operating system, you may need to set the `LIBCLANG_PATH` environment variable or add `llvm-config` to your search path.
+
+### Linux-specific instructions
+
+Set `LIBCLANG_PATH` to the `lib` directory of your `llvm` installation. E.g.,
+`LIBCLANG_PATH=/usr/lib/llvm-3.9/lib`.
+
+### Windows-specific instructions
+
+Ensure that `R_HOME` is an environment variable pointing to the preferred R
```

Comment:
Is the `R_HOME` environment variable actually needed somewhere or can you just set the path directly, without setting `R_HOME`?